Kickstarter-iOS/Views/Controllers/ToggleViewControllerTests.swift

```diff
@@ -0,0 +1,58 @@
+@testable import Kickstarter_Framework
+@testable import Library
+import UIKit
+
+final class ToggleViewControllerTests: TestCase {
+  override func setUp() {
+    super.setUp()
+
+    AppEnvironment.pushEnvironment(mainBundle: Bundle.framework)
+    UIView.setAnimationsEnabled(false)
+  }
+
+  override func tearDown() {
+    AppEnvironment.popEnvironment()
+    UIView.setAnimationsEnabled(true)
+
+    super.tearDown()
+  }
+
+  func testView() {
+    let devices = [Device.phone4_7inch, Device.pad]
+    combos([Language.en], devices).forEach { language, device in
+      withEnvironment(language: language) {
+        let controller = ToggleViewController.instantiate()
+        controller.titleLabel.text = "Title for testing purposes only"
+        controller.toggle.setOn(true, animated: false)
+
+        let (parent, _) = traitControllers(device: device, orientation: .portrait, child: controller)
+
+        parent.view.frame.size.height = 60
+
+        FBSnapshotVerifyView(parent.view, identifier: "lang_\(language)_device_\(device)")
+      }
+    }
+  }
+
+  func testViewSupportsDynamicType() {
```

Comment:
I've also decided to add Dynamic type snapshot in order to keep a regression test for the resizing of the label. This should only be done on the base class (no need for subclasses to handle this)